### DISCUSSION TITLE
Authelia SMTP Error with Proton Bridge

### DISCUSSION THREAD

**Discussion by seandonohoo** [authelia/authelia#4716] [Q&A] [answered]
Hello Authelia community, 

Working on setting up Authelia with Proton mail bridge. I finally think I've ran into a bug. When I try and register a device I get the following output: 
`
time="2023-01-06T19:57:22Z" level=error msg="notifier: smtp: failed to dial connection: SMTP AUTH failed: unexpected server response: 2.0.0 Authentication succeeded" method=POST`

It looks like the authelia client doesn't know how to handle the actual 2.0.0 authentication succeeded error. Is anyone else able to confirm?

**Comment by seandonohoo** [comment]
I'm thinking this is a bug but before I open an issue wanted to get confirmation.

**Comment by james-d-elliott** [comment, marked as answer]
Just saw this sorry for not replying sooner, it's indeed a bug with the SMTP server as far as I can tell, 2.0.0 is not a documented code.
